scripts/main.ts

```diff
@@ -37,34 +37,200 @@ const AutocratIDL: AutocratV0 = require("../target/idl/autocrat_v0.json");
 const OpenbookTwapIDL: OpenbookTwap = require("../tests/fixtures/openbook_twap.json");
 const AutocratMigratorIDL: AutocratMigrator = require("../target/idl/autocrat_migrator.json");
 
-const AUTOCRAT_PROGRAM_ID = new PublicKey(
-  "metaRK9dUBnrAdZN6uUDKvxBVKW5pyCbPVmLtUZwtBp"
+// DAO Tokens
+// DeansList
+export const DEAN_DEVNET = new PublicKey(
+  "DEANPaCEAfW2SCJCdEtGvV1nT9bAShWrajnieSzUcWzh"
+)
+export const DEAN = new PublicKey(
+  "Ds52CDgqdWbTWsua1hgT3AuSSy4FNx2Ezge1br3jQ14a"
+)
+// Future DAO
+export const FUTURE_DEVNET = new PublicKey(
+  "DUMm13RrZZoJAaqr1Tz7hv44xUcrYWXADw7SEBGAvbcK"
 );
-const CONDITIONAL_VAULT_PROGRAM_ID = new PublicKey(
-  "vAuLTQjV5AZx5f3UgE75wcnkxnQowWxThn1hGjfCVwP"
-);
-const OPENBOOK_TWAP_PROGRAM_ID = new PublicKey(
-  "twAP5sArq2vDS1mZCT7f4qRLwzTfHvf5Ay5R5Q5df1m"
-);
-export const OPENBOOK_PROGRAM_ID = new PublicKey(
-  "opnb2LAfJYbRMAHHvqjCwQxanZn7ReEHp1k81EohpZb"
-);
-
+export const FUTURE = new PublicKey(
+  "FUTURETnhzFApq2TiZiNbWLQDXMx4nWNpFtmvTf11pMy"
+)
+// MetaDAO for Mainnet and Devnet
 export const META = new PublicKey(
   "METADDFL6wWMWEoKTFJwcThTbUmtarRJZjRpzUvkxhr"
 );
-export const DEVNET_USDC = new PublicKey(
+// Base Tokens
+// Meta USDC (created for use in our contracts)
+export const DEVNET_MUSDC = new PublicKey(
   "B9CZDrwg7d34MiPiWoUSmddriCtQB5eB2h9EUSDHt48b"
+)
+// Circle Devnet USDC (from faucet)
+export const DEVNET_USDC = new PublicKey(
+  "4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU"
```

Comment:
@jshiohaha this token is just normal token program but USDC which comes from the faucet for USDC https://faucet.circle.com/ is there a way to use original tokenkeg or is that data pulled from the OLD token list etc? It shouldn't be priority, however just was curious what happens if we don't use metaplex?